src/plugins/maps_legacy/public/map/service_settings.test.js

```diff
@@ -42,11 +42,12 @@ import EMS_STYLE_ROAD_MAP_BRIGHT from '../__tests__/map/ems_mocks/sample_style_b
 import EMS_STYLE_ROAD_MAP_DESATURATED from '../__tests__/map/ems_mocks/sample_style_desaturated';
 import EMS_STYLE_DARK_MAP from '../__tests__/map/ems_mocks/sample_style_dark';
 import { ORIGIN } from '../common/constants/origin';
-import { ServiceSettings } from './service_settings';
+import { ServiceSettings, DEFAULT_SERVICE } from './service_settings';
 
 describe('service_settings (FKA tile_map test)', function () {
   const emsFileApiUrl = 'https://files.foobar';
   const emsTileApiUrl = 'https://tiles.foobar';
+  const emsFakeManifestApiUrl = 'https://fakemapmanifest.foobar';
```

Comment:
Looks like this URL is breaking the link checker. Can you either use an ignored URL from `./.lycheeexclude` or add this there. Ideally using one from there is preferred.

Reply:
sure, will update